src/commands/ext-configure.ts

```diff
@@ -115,29 +130,20 @@ export default new Command("ext:configure <extensionInstanceId>")
       manifest.showPreviewWarning();
       return;
     }
-
+    if (!projectId) {
```

Comment:
Hmmm, I think this is actually redundant - we call needProjectId on l144, which will error out if project id is undefined.